governance/votes/Trial Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,191 @@
+|**[OracleJob](https://github.com/makerdao/dss-cron/blob/master/src/OracleJob.sol):**|[0xe717Ec34b2707fc8c226b34be5eae8482d06ED03](https://etherscan.io/address/0xe717ec34b2707fc8c226b34be5eae8482d06ed03)|-|
+
+### Aligned Delegate One-Off Compensation
+
+As described in [this summary](https://forum.makerdao.com/t/june-2023-aligned-delegate-compensation/21310) and authorized by the [Governance Scope](https://mips.makerdao.com/mips/details/MIP113#12-scope-bootstrapping), a total of **229.44 MKR** will be distributed to previously active Aligned Delegates as detailed below if this executive proposal passes.
+
+
+| Delegate | Address | Amount (MKR) |
+|--|------|--|
+|0xDefensor|[0x9542b441d65B6BF4dDdd3d4D2a66D8dCB9EE07a9](https://etherscan.io/address/0x9542b441d65B6BF4dDdd3d4D2a66D8dCB9EE07a9)|29.76|
+|Bonapublica|[0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3](https://etherscan.io/address/0x167c1a762b08d7e78dbf8f24e5c3f1ab415021d3)|29.76|
+|QGov|[0xB0524D8707F76c681901b782372EbeD2d4bA28a6](https://etherscan.io/address/0xb0524d8707f76c681901b782372ebed2d4ba28a6)|29.76|
+|True Name|[0x612f7924c367575a0edf21333d96b15f1b345a5d](https://etherscan.io/address/0x612f7924c367575a0edf21333d96b15f1b345a5d)|29.76|
+|UPMaker|[0xbb819df169670dc71a16f58f55956fe642cc6bcd](https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd)|29.76|
+|vigilant|[0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cb55500601bcce9f4cb0a0a72dc226f61)|29.76|
+|WBC|[0xeBcE83e491947aDB1396Ee7E55d3c81414fB0D47](https://etherscan.io/address/0xebce83e491947adb1396ee7e55d3c81414fb0d47)|20.16|
+|PBG|[0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8d4df847db7ffe0b46af084fe031f7691c6478c2)|9.92|
+|Bandhar|[0xE83B6a503A94a5b764CCF00667689B3a522ABc21](https://etherscan.io/address/0xE83B6a503A94a5b764CCF00667689B3a522ABc21)|7.68|
+|Libertas|[0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f](https://etherscan.io/address/0xe1ebffa01883ef2b4a9f59b587fff1a5b44dbb2f)|7.04|
+|PALC|[0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A](https://etherscan.io/address/0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A)|2.24|
+|Harmony|[0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2](https://etherscan.io/address/0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2)|1.92|
+|Vote Wizard|[0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9e72629df4fcaa2c2f5813fbbdc55064345431b1)|1.6|
+|Navigator|[0x11406a9CC2e37425F15f920F494A51133ac93072](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072)|0.32|
+
+
+
+### Ecosystem Actor DAI Budget Stream
+
+As per [MIP106 Article 7.4.2.1A](https://mips.makerdao.com/mips/details/MIP106#7-4-2-1a-), and ratified in [Amendment Subproposal MIP102c2-SP8](https://vote.makerdao.com/polling/QmdnSKPu#poll-detail), the following DAI budget streams will be created if this executive proposal passes:
```

Comment:
Generally not necessary to give the history + poll of amendments. It's enough to point to the relevant line in the ratified scope.